### PR TITLE
Upgrade to Node 20 version

### PR DESCRIPTION
**Pull request by hernanvicente**
With [GitHub Actions transitioning from Node 16 to 20](https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/), some applications using this module receive warnings. For us, Panda, our statistics application GitHub workflows show the following warnings:

iac_actions_run_warnings

This PR tries to update Node and some packages to bear this problem.

### CODE REVIEW

File: dist/index.js
Comment (littldr) [inline dist/index.js:1]: Could you please remove the dist file from your PR? It will be automatically generated on release.

**Review by littldr — CHANGES_REQUESTED** [review]
Thanks for this contribution, @hernanvicente!

I requested a single change, but I'm happy to merge and release it asap. :)

File: dist/index.js
Comment (hernanvicente) [reply dist/index.js:1]: I removed the folder. It was a leftover that helped me test the update =)

**Comment by hernanvicente** [comment]
Glad to help, one thing more was pending, and I just updated the release workflow. This was using 'JasonEtco/build-and-tag-action@v1', which relies on Node 16. I forked the code and released a version with Node 20. The action now uses the new repository.

**Review by littldr — APPROVED** [review]
Great. Thanks! :)

**Pull request merged**